README.md

```diff
@@ -21,27 +13,44 @@ Ideally, all software needed for a test should be included by building
 it into the image from the SDK.
 
 Kola supports running tests on multiple platforms, currently QEMU, GCE,
-and AWS. In the future systemd-nspawn and other platforms may be added.
+AWS, vmware ESX and Packet. In the future systemd-nspawn and other
+platforms may be added.
 Local platforms do not rely on access to the Internet as a design
 principal of kola, minimizing external dependencies. Any network
 services required get built directly into kola itself. Machines on cloud
 platforms do not have direct access to the kola so tests may depend on
 Internet services such as discovery.etcd.io or quay.io instead.
 
 Kola outputs assorted logs and test data to `_kola_temp` for later
-inspection. This directory is completely wiped and recreated every time.
+inspection.
 
 Kola is still under heavy development and it is expected that its
 interface will continue to change. Both the CLI and test registration
 interface have upcoming changes.
 
+By default, kola uses the `qemu` platform with the most recently built image
+(assuming it is run from within the SDK).
+
 ### kola run
 The run command invokes the main kola test harness. The harness will
 run any registered tests on all platforms unless otherwise specified. It
 runs any tests whose registered names matches a glob pattern.
 
 `kola run <glob pattern>`
 
+To get a list of all tests, run
+
+`kola list`
+
+### kola spawn
+The spawn command launches CoreOS instances. Use `-s` to login to the
+instance spawned.
+
+### kola mkimage
+The mkimage command sets the linux serial console to /dev/ttyS0. This
```

Comment:
"...creates a copy of a `coreos_*_image.bin` with its primary console set to the serial port."